worlds/alttp/Rom.py

```diff
@@ -1730,7 +1732,7 @@ def write_custom_shops(rom, world, player):
             replacement_price_data = get_price_data(item['replacement_price'], item['replacement_price_type'])
             slot = 0 if shop.type == ShopType.TakeAny else index
             if item['player'] and world.game[item['player']] != "A Link to the Past":  # item not native to ALTTP
-                item_code = get_nonnative_item_sprite(item['item'])
```

Comment:
This seems janky but I don't know how else to get an item code from just the name. But also I think a lot of the shop code is janky and should be rewritten at some point.

Reply:
world.item_name_to_id: dict[str, int]